### DISCUSSION TITLE
Support for madoko

### DISCUSSION THREAD

**Discussion by tholzschuh** [srid/emanote#209] [Ideas]
[Madoko](http://madoko.org/reference.html) is an extension of markdown tailored towards making it easy to write scientific papers.

To my knowledge, it provides the best support for properly embedding mathematics within markdown, especially since it's 100% compatible with LaTeX.

I'd really like to set up a Zettelkasten to organize my notes on mathematics, but all the approaches I've seen so far don't have enough support of LaTeX for this to be easily doable (especially things like drawing commutative diagrams etc.).

How difficult would it be to implement full madoko support within emanote?

**Comment by srid** [comment]
I'm not familiar with this. But if Pandoc supports the format, it should be relatively straightforward to add. See also https://github.com/srid/emanote/issues/18

**Reply by tholzschuh** [reply]
Pandoc doesn't support it (yet?), but Madoko rather comes with its own compiler outputting PDF/HTML.
The only 'missing' feature seems to be bidirectional links..
